cards/src/main/resources/cards/custom/group0/other_death_knight/ungoro/minion_bonefetcher.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Bonefetcher",
+  "baseManaCost": 2,
+  "type": "MINION",
+  "heroClass": "SPIRIT",
+  "baseAttack": 2,
+  "baseHp": 3,
+  "rarity": "RARE",
+  "description": "Battlecry: Gain the text of a random minion that was shuffled into your deck.",
```

Comment:
Maybe it would be simpler if this card said

`Battlecry: Gain the text of a random minion in your deck that didn't start there.`